.github/workflows/cov.yaml

```diff
@@ -22,7 +22,8 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: "1.21.x"
+          go-version-file: src/github.com/nats-io/nats-server/go.mod
```

Comment:
@wallyqs Although this is right now using go 1.22 (in go.mod), but if the go.mod does get updated for a while, this may not be the good approach (took that from the mqtt yml file). What do you think, should I be explicitly using "1.23.x" here? (and update as we update our go version requirements?)

Reply:
We're generally keeping `go.mod` up-to-date with support for Go N and N-1 so I think this is probably fine.